### DISCUSSION TITLE
🧩 FEATURE REQUEST: Search bar above articles

### DISCUSSION THREAD

**Discussion by jacobsamo** [dailydotdev/daily#1004] [Feature Requests]
## Feature Request
Add a search bar at the top of the articles for desktop to easily search items from there

### Description

I Would like to have a search bar above the articles to for easy searching

### Solution

Add a small search bar above the article section. with the user possible be able to choose what search engine that they want to use e.g Google.com, You.com, etc. the user could also choose whether they want the search bar or not in the man section

Sketch:
![Sketch](https://user-images.githubusercontent.com/77374574/193949192-a53c47ba-a7e0-48cc-9d9d-ac71c54ac71a.png)

Examples
<img width="358" alt="Google search" src="https://user-images.githubusercontent.com/77374574/193946192-afc09e80-84bb-4b15-8e4c-a20c0e68ba75.png">
<img width="358" alt="BlackBox search" src="https://user-images.githubusercontent.com/77374574/193946202-9590cdf1-8929-4464-a443-c0ee26a27cf2.png">

### Alternate Solutions

Give the user the choice of having a search bar above the articles.

**Comment by idoshamun** [comment]
Hi @Eirfire,

Thanks for the suggestion. We have a dedicated search page to search for daily.dev articles.
We are not planning to add an external search bar at the moment.

**Comment by fabus961** [comment]
Hey @idoshamun, I came accross this FR because I just wanted to create the same one ;)

Let me elaborate the FR story a bit:

As a user I want to have a global search function, so I can browse daily.dev articles _withouth leaving the page._  

Maybe the demand isn't high enough, but it's worth a try ;)

**Comment by idoshamun** [comment]
@fabus961 I'd love to learn more on why you search for a specific article? We're considering revamping our search and have several ideas

**Comment by fabus961** [comment]
@idoshamun Because there was relevant information that you barely remember. So you try to search if you haven't found it. And you start scrolling and scrolling and propably scroll some more to get back in time. Sometimes even stuff in your own bookmarks is kinda hard to dig out again...
